services/web-app/data/libraries.js

```diff
@@ -408,6 +415,13 @@ const devLibraries = {
     group: carbonComponentsGroup,
     ref: 'v10'
   },
+  'carbon-components-vue': {
+    host: 'github.com',
+    org: 'carbon-design-system',
+    repo: 'carbon-components-vue',
+    path: '/packages/core/src',
+    group: carbonComponentsGroup
+  },
```

Comment:
Must specify packageJsonPath since it's not at the same level
```suggestion
  'carbon-components-vue': {
    host: 'github.com',
    org: 'carbon-design-system',
    repo: 'carbon-components-vue',
    path: '/packages/core/src',
    packageJsonPath: '/../package.json',
    group: carbonComponentsGroup
  },
```